tests/auth_tests/test_auth_backends.py

```diff
@@ -667,3 +683,30 @@ def test_backend_path_login_with_explicit_backends(self):
         user = User.objects.create_user(self.username, 'email', self.password)
         self.client._login(user, self.other_backend)
         self.assertBackendInSession(self.other_backend)
+
+
+@override_settings(
+    AUTHENTICATION_BACKENDS=['django.contrib.auth.backends.AllowInactiveUsersModelBackend'],
+)
+class AllowInactiveUsersModelBackendTest(TestCase):
+
+    def setUp(self):
```

Comment:
prefer `setUpTestData` since that executes once per test class instead of once for every method